keyboards/clueboard/2x1800/2019/rules.mk

```diff
@@ -0,0 +1,21 @@
+# MCU details
+MCU = at90usb1286
+F_CPU = 16000000
+ARCH = AVR8
+F_USB = $(F_CPU)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+OPT_DEFS += -DBOOTLOADER_SIZE=1024
+BOOTLOADER = halfkay
```

Comment:
```suggestion
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
BOOTLOADER = halfkay
```

Reply:
Since these are always built with teensy 2.0++'s I figured we didn't need to document the other bootloaders here.

Reply:
I know, but I'd like for it to be aligned with the template. I mentioned in another PR that the docs should already list these, so we ought to just have the `# Bootloader selection` line, but I'd want to update all of the rules.mk's (in chunks, of course).